packages/react-components/src/components/abstract-xy-output-component.tsx

```diff
@@ -269,6 +269,8 @@ export abstract class AbstractXYOutputComponent<P extends AbstractOutputProps, S
                     outputStatus: treeResponse.status,
                     xyTree: treeResponse.model.entries,
                     columns
+                }, () => {
+                    this.updateXY();
```

Comment:
`this._debouncedUpdateXY()` will prevent the chart to display the data progressively while it is being loaded. It will show the first part of it, freeze and wait until all the data is displayed.

Therefore, to have the effect of watching the data being rendered while it is being loaded, `updateXY()` has to be called directly here.

I added a check to avoid calling `updateXY()` from here in charts that do not render any series before the user clicks on any checkboxes.